docs/usage/key-concepts/dashboard.md

```diff
@@ -66,6 +66,29 @@ Here is an example of how this can look:
 | npm        | `airbnb-prop-types` | ![Available](https://img.shields.io/badge/available-green?style=flat-square)      |
 | npm        | `left-pad`          | ![Unavailable](https://img.shields.io/badge/unavailable-orange?style=flat-square) |
 
+#### Abandonment detection
+
+Renovate includes the `abandonments:recommended` preset (automatically included in `config:best-practices`) to help identify potentially abandoned packages. This preset:
+
+- Sets a default [`abandonmentThreshold`](../configuration-options.md#abandonmentthreshold) of `1 year` for all packages
+- Includes community-sourced overrides for packages that appear abandoned but are still maintained
+- Prevents updates to truly abandoned packages while allowing updates to packages with irregular release schedules
+
+**Key overrides:**
+
+- `@types/*` packages (npm): Set to `eternal` - TypeScript definitions don't need frequent releases
+- `lodash` (npm): Set to `6 years` - Maintainers still respond to security issues despite infrequent releases
```

Comment:
This list will require manual updating in docs, so I'm not sure if it's a good idea. Maybe just the `@types/` one?